### PR TITLE
:recycle: Move form to new component

### PR DESCRIPTION
**Pull request by davidwobido**
image

Add new component folder with form component and its css module file

### CODE REVIEW

File: src/app/components/form/Form.module.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  display: grid;
+  background-color: rgb(249 233 194);
+  border-radius: 1rem;
+  padding: 2.5rem;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  row-gap: 1rem;
+  column-gap: 0.5rem;
+  align-items: center;
+  justify-items: center;
```
Comment (leonardpaulus) [inline src/app/components/form/Form.module.css]:
```suggestion
place-items: center;
```

Comment (manusmd) [inline src/app/components/form/Form.module.css]:
```suggestion
  place-items: center;
```

File: src/app/components/form/Form.module.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  grid-column: 1/3;
+  color: rgb(35 103 106);
+  text-transform: uppercase;
+  letter-spacing: 0.1rem;
+  font-size: 2rem;
+}
+
+.form__text {
+  grid-row: 2/3;
+  border: hidden;
```
Comment (manusmd) [inline src/app/components/form/Form.module.css]: Why not like that?
```suggestion
  border: none;
```

Comment (leonardpaulus) [inline src/app/components/form/Form.module.css]: I can't seem to find anything for "border: hidden". Did you mean"border: none"?

File: src/app/components/form/Form.module.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.form__submit {
+  grid-row: 3/4;
+  color: white;
+  background-color: rgb(35 103 106);
+  grid-column: 1/3;
+  padding: 0.75rem 1.5rem;
+  border: hidden;
+  border-radius: 10px;
```
Comment (leonardpaulus) [inline src/app/components/form/Form.module.css]:
```suggestion
  border-radius: .625rem;
```

**Review by leonardpaulus — APPROVED** [review]
Looks great, just some minor suggestions <3

**Review by manusmd — CHANGES_REQUESTED** [review]
lgtm 🔥 Just a small change

**Review by JulianSchumacher99 — APPROVED** [review]
lgtm 🔥

**Review by fabiglahn — COMMENTED** [review]
🚀

File: src/app/components/form/Form.module.css (hunk 2 above)
Comment (davidwobido) [reply src/app/components/form/Form.module.css]: it’s the same as „border-style: hidden“

**Pull request merged**